### PR TITLE
Addition of more Registries

### PR DESCRIPTION
**Pull request by jpradocueva**
The following Registries were added:
- OMA Security
- OMA Device Management

### CODE REVIEW

File: OMNA/registerpage.html [hunk 1]
```diff
@@ -288,6 +288,600 @@
             </table>
         </div>
 
+        <div>
+            <h2>OMA Security</h2>
+            <table id="registries-table" class="table-bordered">
```
Comment (nathan-omaorg) [inline OMNA/registerpage.html]: I think you are overusing "id".

id should be unique on a page. You have many "registries-table" id's.

There should also be a reason for giving an element an id. Such as for styling, or for Javascript.

Please remove any id's that are not unique, or have no use.

Thank you

File: OMNA/registerpage.html [hunk 2]
```diff
@@ -288,6 +288,600 @@
...
+                <thead>
+                    <tr>
+                        <th>Registry Type</th>
+                        <th>Information Requested</th>
+                        <th>Action</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td id="iso-object-identifiers">
```
Comment (nathan-omaorg) [inline OMNA/registerpage.html]: Are all these td id's used for something?

Comment (github-oma) [inline OMNA/registerpage.html]: @Nathan, yes they are used to link to a particular register from "Registries.hmtl" page. I haven't done yet the PR to do the link yet.
Is this 'ID' set correctly as part of the ``<td>`` ?

File: OMNA/registerpage.html (hunk 1 above)
Comment (github-oma) [inline OMNA/registerpage.html]: all of them are unique and are for a use. I will wait to create my second Pull Request to link to these IDs after we have a chat tomorrow.

**Pull request merged**